Library/Formula/snakemake.rb

```diff
@@ -0,0 +1,22 @@
+# coding: utf-8
```

Comment:
Oh, strange. It might be that Poet hasn't been tweaked to not add that line yet. @tdsmith is the creator of that tool so will defer to him, but as far as I'm aware it shouldn't be needed any more.